vernac/vernacstate.mli

```diff
@@ -32,3 +32,53 @@ val make_shallow : t -> t
 
 (* WARNING: Do not use, it will go away in future releases *)
 val invalidate_cache : unit -> unit
+
+(* Compatibility module *)
```

Comment:
I think the comment should say explicitly that this should not be used.

Reply:
Fixed thanks.